python/src/diffusionkit/mlx/mmdit.py

```diff
@@ -228,18 +235,17 @@ def __call__(
             for bidx, block in enumerate(self.unified_transformer_blocks):
                 latent_unified_embeddings = block(
                     latent_unified_embeddings,
-                    timestep,
+                    timestep_embedding,
                     positional_encodings=positional_encodings,
                 )
 
             latent_image_embeddings = latent_unified_embeddings[
                 :, token_level_text_embeddings.shape[1] :, ...
             ]
 
-        # Final layer
         latent_image_embeddings = self.final_layer(
             latent_image_embeddings,
-            timestep,
+            timestep_embedding
```

Comment:
Override original timestep only for dev model

```suggestion
            timestep,
```